drone/exec.go

```diff
@@ -141,6 +149,7 @@ func execCmd(c *cli.Context) error {
 
 	var exits []int
 
+axes:
```

Comment:
any reasons for the `axes` label here? I don't see a breakpoint or goto anywhere